source/mobile/build-custom-mobile-apps.rst

```diff
@@ -0,0 +1,84 @@
+
+In general, building your own Mobile apps will present some challenges, including:
+
+- Installing the necessary developer tools (such as Nodejs, XCode Developer Tools, Android SDKs, as well as others).
+- Obtaining and providing certificates for your custom Mattermost Mobile apps*.
+- Signing your custom Mattermost Mobile apps *.
+- Distributing your Mobile app to your users.
+
+* Mattermost Mobile apps are signed and have certificates associated with Mattermost and public app stores. This means Mattermost’s Mobile App won’t work if you choose to privately host the Mattermost Push Proxy Service (MPNS). You'll need to build your own custom mobile app.
+
+Custom Mobile App Branding
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Ensure you select a unique app name that helps users distinguish your version from others, such as "<Your Company Name> Chat". See our `Brand and Visual Design Guidelines <https://handbook.mattermost.com/operations/operations/company-processes/publishing/publishing-guidelines/brand-and-visual-design-guidelines#name-usage-guidelines>`__ in our Company Handbook for branding details.
+
+Deploying Your Custom App
+-------------------------
+
+You have a number of ways to deploy your custom app for your users to download to their mobile devices.
+
+Using an Enterprise App Store
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This is the most common way for customers to distribute their apps, and it’s our recommended approach. Once your custom app is added to your own Enterprise App Store, your users can download it from the store directly or from an EMM catalog. 
+
+Alternatively, you can use an EMM provider to push the application to the user’s device, then use the AppConfig standard to enforce a selection of app-specific controls. See our `Manage App Configuration Using AppConfig <https://docs.mattermost.com/mobile/deploy-mobile-apps-using-emm-provider.html#manage-app-configuration-using-appconfig>`__ documentation for details.
+
+Using an EMM Provider
+~~~~~~~~~~~~~~~~~~~~~
+
+See our `Deployment Using an EMM Provider <https://docs.mattermost.com/mobile/deploy-mobile-apps-using-emm-provider.html>`__ documentation to learn more about deploying your custom Mattermost Mobile apps through an EMM provider.
+
+Using Public App Stores
+~~~~~~~~~~~~~~~~~~~~~~~
+
+This is the same process Mattermost uses to make Mattermost Mobile apps available for everyone. It’s a good choice if you're looking to white-label your own Mobile apps to remove Mattermost branding. To submit an app to the public app stores, you need to submit the app to public app stores for review and approval.
+
+As part of the submission process, you need to identify an update strategy that accounts for the release of new versions of Mattermost Mobile apps. 
+
+This update strategy should include:
+
+- Reviewing compatibility requirements.
+- Validating mobile app versions connecting to the server.
+- Updating Mattermost Server.
```

Comment:
Are we referring to the binary or the physical server?

Reply:
That's an excellent question. @esethna - Can you confirm?

Reply:
binary